packages/block-library/src/post-title/edit.js

```diff
@@ -1,10 +1,32 @@
+/**
+ * External dependencies
+ */
+import classnames from 'classnames';
+
 /**
  * WordPress dependencies
  */
 import { useSelect } from '@wordpress/data';
+import {
+	AlignmentToolbar,
+	BlockControls,
+	__experimentalBlock as Block,
+} from '@wordpress/block-editor';
+import { ToolbarGroup } from '@wordpress/components';
 
-export default function PostTitleEdit( { context } ) {
+/**
+ * Internal dependencies
+ */
+import HeadingLevelDropdown from '../heading/heading-level-dropdown';
```

Comment:
Unsure of the best approach to extract this as a shared component.

Reply:
I think it's reasonable, at some point we might consider whether it's something we want to make available for third-party block authors and move to "block-editor".